framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -5824,8 +5823,7 @@ VkResult VulkanReplayConsumerBase::OverrideCreateShaderModule(
 
             if (options_.dumping_resources)
             {
-                VulkanShaderModuleInfo* shader_info =
-                    reinterpret_cast<VulkanShaderModuleInfo*>(pShaderModule->GetConsumerData(0));
+                auto* shader_info = reinterpret_cast<VulkanShaderModuleInfo*>(pShaderModule->GetConsumerData(0));
```

Comment:
`auto` should be enough

Reply:
done

Reply:
even though.. https://releases.llvm.org/18.1.6/tools/clang/tools/extra/docs/clang-tidy/checks/readability/qualified-auto.html

Reply:
yeah, we currently have a rather fragmented state wrt. that. maybe we should come back to this with a more targeted refactoring-PR. there are other examples reported by clang-tidy, like e.g. redundant usage of virtual/override keywords, empty default-ctors, etc.

Reply:
I suppose we might prioritize some checks that are more valuable than others in finding potential bugs.

Reply:
so far that's mostly `-Wshadow` for me. many remarks from clang-tidy are in fact more about "tidiness" or even just cosmetic.